packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -132,6 +132,120 @@ void main() {
     expect(material.elevation, elevation);
     expect(material.shape, shape);
   });
+
+  testWidgets('BottomSheet persistentElevation takes priority over elevation - Persistent', (WidgetTester tester) async {
+    const modalElevation = 5.0;
+    const persistentElevation = 7.0;
+    const bottomSheetTheme = BottomSheetThemeData(
+      elevation: modalElevation,
+      persistentElevation: persistentElevation,
+    );
+    await tester.pumpWidget(bottomSheetWithElevations(bottomSheetTheme));
+    await tester.tap(find.byIcon(Icons.check));
+    await tester.pumpAndSettle();
+    final Material material = tester.widget<Material>(
+      find.descendant(
+        of: find.byType(BottomSheet),
+        matching: find.byType(Material),
+      ).first,
+    );
+    expect(material.elevation, persistentElevation);
+  });
+
+  testWidgets('BottomSheet persistentElevation takes priority over elevation - Modal', (WidgetTester tester) async {
```

Comment:
I think this should say elevation takes priority over persistentElevation

Reply:
over modal elevation

Reply:
Done